src/components/vertical-menu/vertical-menu.stories.tsx

```diff
@@ -320,3 +322,95 @@ export const FullScreen: Story = () => {
   );
 };
 FullScreen.storyName = "Full Screen";
+
+export const FullScreenWithModal: Story = () => {
```

Comment:
**issue:** It appears in this example that there is an issue with the keyboard navigation. If you open the menu with the keyboard, the focus still seems to be on the `Confirm` component behind the menu. Drop me a message if you struggle to recreate this, and I can do a screen recording and send it over to you 😄